frontend/src/common/authentication/auth-walls/IDIRAuthWall.tsx

```diff
@@ -84,7 +84,7 @@ export const IDIRAuthWall = ({
     }
 
     const doesUserHaveAccess = DoesUserHaveAuthGroup<IDIRUserAuthGroupType>({
-      userAuthGroup: idirUserDetails?.userAuthGroup,
+      userAuthGroup: idirUserDetails?.userRole,
       allowedAuthGroups,
```

Comment:
Should we still have the key named "userAuthGroup" instead of "userRole"? Similarly for "allowedAuthGroups" -> "allowedRoles"